internal/handlers/common/error.go

```diff
@@ -122,26 +122,25 @@ func ProtocolError(err error) (*Error, bool) {
 func formatBitwiseOperatorErr(err error, operator string, maskValue any) error {
 	switch err {
 	case errNotWholeNumber:
-		return NewErrorMsg(
-			ErrFailedToParse,
+		return NewErrorMsg(ErrFailedToParse,
```

Comment:
We don't write
```go
func foo() { var a int
var b string
```

That's the same thing

Reply:
That was a code change to make linters pass.

Reply:
WDYM?

Reply:
https://github.com/FerretDB/FerretDB/pull/531#discussion_r858360983

Reply:
then maybe can we change linter rules to make it impossible to write in the other style? 
I run linters locally and it was complaining.

Reply:
> I run linters locally and it was complaining.

So did I. So I changed those line. And you changed those lines in parallel. But my PR was merged first, so now that PR changes lines that do not trigger linter already.

> then maybe can we change linter rules to make it impossible to write in the other style?

Once it works – sure. The only problem is that golangci-lint mostly does not work now due to generics